helm/provider_test.go

```diff
@@ -59,10 +61,18 @@ func testAccPreCheck(t *testing.T, namespace string) {
 		createNamespace(t, namespace)
 	}
 
-	os.Setenv("HELM_REPOSITORY_CONFIG", "/Users/amell/Library/Preferences/helm/repositories.yaml")
-	os.Setenv("HELM_REPOSITORY_CACHE", "/Users/amell/Library/Caches/helm/repository")
-	os.Setenv("HELM_REGISTRY_CONFIG", "/Users/amell/Library/Preferences/helm/registry.json")
-	os.Setenv("HELM_PLUGINS", "/Users/amell/Library/helm/plugins")
+	helmdir, err := ioutil.TempDir(os.TempDir(), "helmhome")
+	defer os.RemoveAll(helmdir)
```

Comment:
This deferred cleanup doesn't work as expected. The tempdir is being removed right after the function call returns and then helm re-creates the same path and the resulting files never get cleaned up.
I'm looking into a better solution to manage the mess.